src/fromager/commands/download_sequence.py

```diff
@@ -47,32 +54,39 @@ def download_sequence(
         wheel_servers = [sdist_server_url]
 
     with open(build_order_file, "r") as f:
-        for entry in progress.progress(json.load(f)):
-            if entry["prebuilt"]:
-                logger.info(f"{entry['dist']}: uses a pre-built wheel, skipping")
-                continue
+        build_order = json.load(f)
+
+    def download_one(entry):
```

Comment:
Could you add the types here

Reply:
Done, although it's a vague one. Do we have an alias for the entry in a build-order file?

Reply:
Its on my todo list for the types cleanup

Reply:
OK. I've added at least the `dict` part here.